eslint.config.mjs

```diff
@@ -13,7 +13,7 @@ const __dirname = dirname(__filename);
  * @param {string | undefined} pathGlob
  * @returns {string}
  */
-const addPrefix = (prefix, pathGlob = '**/*') => pathGlob.replace(/^(!?)(\.?\/)?/, `$1${prefix}/`);
+const addPrefix = (prefix, pathGlob = '**/*') => pathGlob.replace(/^(!?)(\.?\/)?/, `$1${prefix}/`).replaceAll(sep, posix.sep);
```

Comment:
Need to change it in the template file of the ng-add @o3r/eslint-config

(sorry I forgot it before you pushed) :)